test/cql-pytest/cassandra_tests/validation/operations/alter_test.py

```diff
@@ -101,7 +101,7 @@ def testDropAddWithDifferentKind(cql, test_keyspace):
         assert_invalid_message(cql, table, "Cannot re-add previously dropped column 'd' of kind REGULAR, incompatible with previous kind STATIC",
```

Comment:
Please add a test that checks using bind variables. It's the first schema altering statement with bind variables, so if something wasn't wired correctly, we won't know.

I think it's impossible to get it wrong, but better to test.

Reply:
Ah, you already have it.